vacasa/connect/connect.py

```diff
@@ -885,6 +944,124 @@ def add_identity_check(self,
 
         return self._post(url, json={'data': {'attributes': payload,
                                               'type': 'reservation-guest'}}, headers=headers).json()
+    """
+        https://vacasa.docs.stoplight.io/contracts/get-contracts-list
+    """
+    def get_contracts(self,
+                      params: dict= None,
+                      active_only: bool=True):
+        if params is None:
+            params = {}
+
+        if not active_only:
+            params['filter[terminated]'] = False
+
+        url = f"{self.endpoint}/v1/contracts"
+        headers = self._headers()
+
+        return self._iterate_pages(url, headers, params)
+
+    """
+        https://vacasa.docs.stoplight.io/contracts/postv1contracts
+        
+        Args:
+            owners: list of objects that contain three properties 'percentage_ownership', 'tax_ownership', 'contact_id' 
+           
+    """
+    def create_contract(self,
+                        unit_id: int,
+                        management_fee: int,
+                        owners: list,
+                        created_by: int,
+                        start_date: str,
+                        end_date: str= '01/01/2099',
+                        monthly_rent: int=0,
+                        template_version_id: int=1,
+                        form_id: int= 1,
+                        channel_fee_cost_sharing_id: int= 5,
+                        amendment_by_notice_id: int= 4,
```

Comment:
Might be nice to break these magic ints out into `Enum`s

Reply:
Once the connect API documentation is updated, I can update these magic numbers.